apollo-backend/src/test/java/io/logz/apollo/KubernetesHandlerTest.java

```diff
@@ -270,31 +284,37 @@ private static void setMockDeploymentStatus(RealDeploymentGenerator realDeployme
 
         DeploymentStatus deploymentStatus;
         if (finished) {
-            deploymentStatus = new DeploymentStatus(1, 1L, totalReplicas, 0, 1);
+            deploymentStatus = new DeploymentStatus(1, 1, new ArrayList<>(), 1L, 1, totalReplicas, 0, 1);
```

Comment:
The object have changed? does that not affect the deployment status checks?

Reply:
I changed where it has affected